File: ni/ni/ruby/request.rb
require "net/http"
require "uri"

uri = URI.parse("https://rest.nexmo.com/ni/json")
params = {
    "api_key" => API_KEY,
    "api_secret" => API_SECRET,
    "number" => YOUR_NUMBER,
    "callback" => YOUR_CALLBACK
}

response = Net::HTTP.post_form(uri, params)

puts response.body

File: ni/ni/python/request.py
import urllib

params = {
    'api_key': API_KEY,
    'api_secret': API_SECRET,
    'number': YOUR_NUMBER,
    'callback': YOUR_CALLBACK
}

url = 'https://rest.nexmo.com/ni/json?' + urllib.urlencode(params)

response = urllib.urlopen(url)
print response.read()
